charts/linkerd-control-plane/values.yaml

```diff
@@ -324,11 +324,12 @@ networkValidator:
   # @default -- plain
   logFormat: plain
   # -- Address to which the network-validator will attempt to connect. This should be an IP
-  # that the cluster is expected to be able to reach but a port it should not, e.g., a public IP 
+  # that the cluster is expected to be able to reach but a port it should not, e.g., a public IP
   # for public clusters and a private IP for air-gapped clusters with a port like 20001.
-  connectAddr: "1.1.1.1:20001"
+  # If empty, defaults to 1.1.1.1:20001 and [fd00::1]:20001 for IPv4 and IPv6 respectively.
+  connectAddr:
```

Comment:
Fair enough, I'll change it to an explicit empty string for consistency, and to not give the impression this could hold something else besides a string.